shotover-proxy/src/message/mod.rs

```diff
@@ -313,27 +319,47 @@ pub enum Value {
     None,
     #[serde(with = "my_bytes")]
     Bytes(Bytes),
+    Ascii(String),
     Strings(String),
-    Integer(i64),
-    Float(f64),
+    Integer(i64, IntSize),
+    Double(f64),
+    Float(f32),
     Boolean(bool),
     Inet(IpAddr),
     List(Vec<Value>),
     Rows(Vec<Vec<Value>>),
     NamedRows(Vec<HashMap<String, Value>>),
     Document(HashMap<String, Value>),
     FragmentedResponse(Vec<Value>),
+    Set(Vec<Value>),
+    Map(Vec<(Value, Value)>),
+    Varint(BigInt),
+    Decimal(BigDecimal),
+    Date(i32),
+    Timestamp(i64),
+    Timeuuid(Uuid),
+    Varchar(String),
+    Uuid(Uuid),
+    Time(i64),
+    Counter(i64),
+    Tuple(Vec<Value>),
+    Udt(HashMap<String, Value>),
```

Comment:
Are we doing this like field_name maps to value??

Reply:
Yeah